examples/Migration/CampaignManagement/CreateCompleteCampaignBothApisPhase1.php

```diff
@@ -19,10 +19,427 @@
+    }
+
+    /**
+     * Creates a campaign.
+     *
+     * @param AdWordsServices $adWordsServices the AdWords services
+     * @param AdWordsSession $adWordsSession the AdWords session
+     * @param int $budgetId the campaign budget ID
+     * @return Campaign the newly created campaign
+     */
+    private static function createCampaign(
+        AdWordsServices $adWordsServices,
+        AdWordsSession $adWordsSession,
+        int $budgetId
+    ) {
+        /** @var CampaignService $campaignService */
+        $campaignService = $adWordsServices->get($adWordsSession, CampaignService::class);
+
+        // Create a campaign.
+        $campaign = new Campaign();
+        $campaign->setName('Interplanetary Cruise #' . uniqid());
+        $campaign->setAdvertisingChannelType(AdvertisingChannelType::SEARCH);
+        // Recommendation: Set the campaign to PAUSED when creating it to stop
+        // the ads from immediately serving. Set to ENABLED once you've added
+        // targeting and the ads are ready to serve.
+        $campaign->setStatus(CampaignStatus::PAUSED);
+
+        // Set bidding strategy.
+        $biddingStrategyConfiguration = new BiddingStrategyConfiguration();
+        $biddingStrategyConfiguration->setBiddingStrategyType(
+            BiddingStrategyType::MANUAL_CPC
+        );
+        $campaign->setBiddingStrategyConfiguration($biddingStrategyConfiguration);
+
+        // Set the budget.
+        $campaign->setBudget(new Budget());
+        $campaign->getBudget()->setBudgetId($budgetId);
+
+        // Set the campaign network options.
+        $networkSetting = new NetworkSetting();
+        $networkSetting->setTargetGoogleSearch(true);
+        $networkSetting->setTargetSearchNetwork(true);
+        $campaign->setNetworkSetting($networkSetting);
+
+        // Optional: Set the start and end dates.
+        $campaign->setStartDate(date('Ymd', strtotime('+1 day')));
+        $campaign->setEndDate(date('Ymd', strtotime('+1 year')));
+
+        // Create a campaign operation.
+        $operation = new CampaignOperation();
+        $operation->setOperand($campaign);
+        $operation->setOperator(Operator::ADD);
+        $operations[] = $operation;
+
+        // Create the campaign on the server and print out its information.
+        $result = $campaignService->mutate([$operation]);
+        $createdCampaign = $result->getValue()[0];
+        printf(
+            "Campaign with ID %d and name '%s' was created.%s",
+            $createdCampaign->getId(),
+            $createdCampaign->getName(),
+            PHP_EOL
+        );
+
+        return $createdCampaign;
+    }
+
+    /**
+     * Gets a campaign.
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param string $customerId the client customer ID without hyphens
+     * @param string $resourceName the resource name of the campaign to retrieve
+     * @return Campaign the campaign
+     */
+    private static function getCampaign(
```

Comment:
I don't think you need getCampaign or any getter when you use AdWords API to create an object.

Reply:
Done.